cookbook/serverless-customer-service-bot.mdx

```diff
@@ -0,0 +1,296 @@
+1. __Ease of Use:__ Forget the complexities of infrastructure management. With Inferless, you simply bring your model, and within minutes, you have a working endpoint. Deployment is hassle-free, without the need for in-depth knowledge of scaling or infrastructure maintenance.
+2. __Cold-start Times:__ Inferless's unique load balancing ensures faster cold-starts. Expect around 2.87 seconds to process each queries, significantly faster than many traditional platforms.
+3. __Cost Efficiency:__ Inferless optimizes resource utilization, translating to lower operational costs. Here's a simplified cost comparison:
+
+### Scenario 1
+You are looking to deploy a Customer Service Voicebot application for processing 100 queries.<br />
+
+__Parameters:__
+- __Total number of queries:__ 100 daily.<br />
+- __Inference Time:__ All models are hypothetically deployed on A100 80GB, taking 2.87 seconds of processing time and a cold start overhead of 24.01 seconds.<br />
+- __Scale Down Timeout:__ Uniformly 60 seconds across all platforms, except Hugging Face, which requires a minimum of 15 minutes. This is assumed to happen 100 times a day.<br />
+
+__Key Computations:__
+1. __Inference Duration:__ <br/>
+Processing 100 queries and each takes 2.87 seconds<br/>
+Total: 100 x 2.87 = 287 seconds (or approximately 0.08 hours)
+2. __Idle Timeout Duration:__<br/>
+Post-processing idle time before scaling down: (60 seconds - 2.87 seconds) x 100 = 5713 seconds (or 1.59 hours approximately)<br/>
+3. __Cold Start Overhead:__<br/>
+Total: 100 x 24.01 = 2401 seconds (or 0.67 hours approximately)<br/>
+
+__Total Billable Hours with Inferless:__ 0.08 (inference duration) + 1.59 (idle time) + 0.67 (cold start overhead)  = 2.34 hours<br/>
+__Total Billable Hours with Inferless:__ 2.34 hours<br/>
+
+### Scenario 2
+You are looking to deploy a Customer Service Voicebot application for processing 1000 queries per day.<br />
+
+__Key Computations:__<br />
+1. __Inference Duration:__<br />
+Processing 1000 queries and each takes 2.87 seconds
+Total: 1000 x 2.87 = 2870 seconds (or approximately 0.8 hours)‍
+2. __Idle Timeout Duration:__<br/>
+Post-processing idle time before scaling down: (60 seconds - 2.87 seconds) x 100 = 5713 seconds (or 1.59 hours approximately)<br/>
+3. __Cold Start Overhead:__<br/>
+Total: 100 x 24.01 = 2401 seconds (or 0.67 hours approximately)<br/>
+
+__Total Billable Hours with Inferless:__ 0.8 (inference duration) + 1.59 (idle time) + 0.67 (cold start overhead)  = 3.06 hours<br/>
+__Total Billable Hours with Inferless:__ 3.06 hours<br/>
+
+
+| Scenarios | On-Demand Cost | Serverless Cost|
+| :--- | :---- | :---- |
+|  100 requests/day | \$28.8 (24 hours billed at $1.22/hour) | \$2.85 (2.34 hours billed at $1.22/hour) |
+|  1000 requests/day | \$28.8 (24 hours billed at $1.22/hour) | \$3.73 (3.06 hours billed at $1.22/hour) |
+
+By opting for Inferless, **_you can achieve up to 163.98% cost savings._**<br/>
```

Comment:
@rbgo404 % looks wrong here

Reply:
I am taking the smallest number i.e 2.85. And comparing with 28.8.
Here's a SS from calculator:
<img width="468" alt="image" src="https://github.com/AGInfer/docs/assets/150957746/2783bd59-df96-4a4f-89df-e4781d2e5065">

Reply:
wrong formula rajdeep..